opendistro-elasticsearch-sql.release-notes

```diff
@@ -1,10 +1,12 @@
-## 2019-12-10, Version 1.4.0 (Current)
+## 2020-1-24, Version 1.4.0 (Current)
 
 ### Notable changes
 
 * Feature [#312](https://github.com/opendistro-for-elasticsearch/sql/issues/312): OD 1.4 Release components
 * Feature [#283](https://github.com/opendistro-for-elasticsearch/sql/pull/283): Added github action to build and run tests
 * Feature [#302](https://github.com/opendistro-for-elasticsearch/sql/pull/302): Documentation for basic usage of plugin
+* Feature [#334](https://github.com/opendistro-for-elasticsearch/sql/pull/334): Change the default response format to JDBC
```

Comment:
np but would it be better if we make the order of the changes from later ones to the older ones, since the preview release notes were that order?

Reply:
later ones to older ones? do you mean later -> newer or newer -> older?

Reply:
sorry to confuse you, newer -> order, with a descending order of the issue#/pr#